cockatrice/src/deck_loader.cpp

```diff
@@ -288,13 +289,25 @@ QString DeckLoader::getCardZoneFromName(QString cardName, QString currentZoneNam
     return currentZoneName;
 }
 
-QString DeckLoader::getCompleteCardName(const QString cardName) const
+QString DeckLoader::guessCompleteCardName(const QString cardName) const
 {
+    static const QRegularExpression evenSimplerExp("(\\s+|\\/\\/.*)"); // matches whitespace or double names
```

Comment:
Your regex is not intuitive, and the name doesn't help its cause

Reply:
it simplifies the card names further than the simplify cardname function does, and it matches all whitespace or anything after and including a double slash